app/scripts/controllers/main.js

```diff
@@ -41,6 +41,11 @@ angular.module('moodCatApp')
     $scope.moods = ['Angry', 'Nervous', 'Exciting', 'Happy', 'Pleasing', 'Relaxing',
       'Peaceful', 'Calm', 'Sleepy', 'Bored', 'Sad'];
 
+	$scope.chooseMood = function(mood) {
+		console.log("Choosemood called: " + mood);
+		angular.element("body").css("background-image", "url(http://moodcat.me/mood-bg/" + mood.toLowerCase() + ")");
```

Comment:
Shouldn't we make a class of this? Then you can set the class of the object and in CSS declare the correct url. @GijsWeterings